src/components/@shared/FormInput/InputElement.tsx

```diff
@@ -87,6 +87,23 @@ export default function InputElement({
         />
       )
     case 'radio':
+      return (
+        <div className={styles.radioWrap}>
+          <input
+            className={styles[type]}
+            type={type}
+            name={name}
+            checked={props.defaultChecked}
+            {...props}
+            readOnly
+          />
+          <label
+            className={cx({ [styles.radioLabelPricing]: true, [size]: size })}
```

Comment:
this is introducing a generic component so should not be bound specific to the place it is being used. So should be just `radioLabel` but as mentioned above might not be needed to use extra markup here

Reply:
this class been removed with component. 👍🏽